usb/esp_tinyusb/Kconfig

```diff
@@ -6,6 +6,19 @@ menu "TinyUSB Stack"
         help
             Specify verbosity of TinyUSB log output.
 
+    choice TINYUSB_RHPORT
+        depends on IDF_TARGET_ESP32P4
+        prompt "TinyUSB PHY"
+        default TINYUSB_RHPORT_HS
+        help
+            Allows set the USB PHY Controller for TinyUSB: HS (USB OTG2.0 PHY for HighSpeed) or FS (USB OTG1.1 PHY for FullSpeed).
+            USB OTG2.0 PHY Controller pad numbers: DM - 50, DP - 51
+            USB OTG1.1 PHY Controller pad numbers: DM - 53, DP - 54
+             
+        config TINYUSB_RHPORT_HS
+            bool "HS"
+    endchoice
```

Comment:
Suggest leaving out any references to selection of PHY for now. Our initial only supports the HS OTG controller using the internal UTMI PHY. Once we figure out the PHY selection logic in the future, we can add options such as this.